Installation/cmake/modules/CGAL_add_test.cmake

```diff
@@ -32,6 +32,9 @@ if(CGAL_RUN_TESTS_THROUGH_SSH)
   find_program(scp_executable scp)
 endif()
 
+if(CGAL_WIN32_CMAKE_ON_CYGWIN)
+  find_program(MSBUILD_CMD NAMES msbuild.exe)
+endif()
```

Comment:
CMake-on-Cygwin is not the right way to test.

1. CYgwin cannot be mandatory to run CTest on Windows.
2. There are other compilers on Windows than `cl.exe`.

You should use the variable [`CMAKE_VS_MSBUILD_COMMAND`](https://cmake.org/cmake/help/latest/variable/CMAKE_VS_MSBUILD_COMMAND.html). That variable is only defined by Visual Studio generators, and that is exactly for those generators we need another way to test.